frontend/src/views/Pages/AccountOverview/GddSend.vue

```diff
@@ -226,7 +244,8 @@ export default {
       },
       send: false,
       row_check: false,
-      row_thx: false,
+      row_thx: true,
+      row_error: false,
```

Comment:
This looks really fishy. Mind have a look @Mogge

Reply:
das ist jetzt erstmal vorrübergehend. das kann alles viel schmaler und besser gemacht werden. am ende sind das alles dinge die dann später in das einheitliche request modal oder so kommen kann.